src/utils/paging.ts

```diff
@@ -50,5 +30,9 @@ export const fillPage = async <T extends { readonly id: string }>(
     O.map(e => e.id),
     O.toUndefined
   );
-  return { hasMoreResults, items, next, prev };
+  return {
+    items,
+    next,
+    prev
+  };
```

Comment:
Why moving out hasMoreResults from here?

Reply:
Because we don't need it. If `hasMoreResults === false` then `next === undefined`.

Reply:
Explained: if do not have a `next` there are no more results.